### DISCUSSION TITLE
Missing file extension after autocompletion

### DISCUSSION THREAD

**Discussion by dzintars** [zk-org/zk-nvim#55] [Help] [answered]
How do I control markdown link file extension `.md` appearance when autocomplete a link?
Currently `.md` extension for me is stripped off the `[note](202010161930)` which breaks cross-compatibility with Obsidian and GitHub. So I should manually add it to make it `[note](202010161930.md)`
I fail to find the appropriate config toggle.

**Comment by mickael-menu** [comment, marked as answer]
In your `zk` config file, you can [disable dropping the file extension](https://github.com/mickael-menu/zk/blob/main/docs/note-format.md#markdown):

```toml
[format.markdown]
link-drop-extension = false
```

I'll consider making this the default, as interoperability is important.

**Reply by dzintars** [reply]
Thank You! I feel silly. :)
